tests/foreman/ui/test_dashboard.py

```diff
@@ -638,13 +640,12 @@ def test_positive_content_host_subscription_status(self):
             'content-view-id': content_view.id,
             'lifecycle-environment-id': env.id,
             'activationkey-id': activation_key.id,
-        })
+        }, force_use_cdn=True)
```

Comment:
True. There's not much we can do about it - either introduce a workaround (which you wanted to) or wait till next cdn build.